### PR TITLE
Add potato-soup ala Bernhard

### PR DESCRIPTION
**Pull request by bmwiedemann**

### CODE REVIEW

File: recipes/soups/potato-soup.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+|400 ml | Water | Wasser
+|4 g / 0.4 pc | Stock cube | Brühwürfel
+|1 g / 0.5 leaf | Laurel | Lorbeer
+|5 ml | Paprika powder | Paprikapulver
+|300 g | Potatoes | Kartoffeln
+|50 g | Onion | Zwiebel
+|50 g | Wiener sausage | Wiener Würstchen oder Cabanossi
+|150 g | Red Pepper/Paprika | Rote Paprika
+|50 g | Heavy cream cheese | Doppelrahmkäse
```
Comment (dcermak) [inline recipes/soups/potato-soup.adoc:26]: Cream cheese is "Frischkäse" which I assume you're not putting into the soup?

File: recipes/soups/potato-soup.adoc [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+=== Potato Soup
+
+.Ingredients
+[width="80%",cols="2,6,6",options="header"]
+|=========================================================
+|Metric | Ingredient | German/Zutat
+
+|400 ml | Water | Wasser
+|4 g / 0.4 pc | Stock cube | Brühwürfel
+|1 g / 0.5 leaf | Laurel | Lorbeer
```
Comment (dcermak) [inline recipes/soups/potato-soup.adoc]:
```suggestion
|1 g / 0.5 leaf | Bay leaf | Lorbeer
```

**Comment by dcermak** [comment]
Could you also please add a reference to `recipes.adoc`?

**Comment by rsimai** [comment]
I believe you missed to edit recipes.adoc and add your recipe to make it finally appear. Looks tasty!

File: recipes/soups/potato-soup.adoc (hunk 1 above)
Comment (bmwiedemann) [reply recipes/soups/potato-soup.adoc:26]: I probably do:

https://lebensmittel-warenkunde.de/lebensmittel/milch-milchprodukte/kaese/frischkaese-und-speisequark/doppelrahmfrischkaese.html says

> dient der Doppelrahmfrischkäse als ideale Zutat für Soßen, **Suppen, Eintöpfe**, Aufläufe, Fondues und für Gebäck.

**Review by dcermak — APPROVED** [review]
Looks good & tasty to me!

**Comment by dcermak** [comment]
@bmwiedemann can you please resolve the conflicts in `recipes.adoc`?

**Pull request merged**